genic/params.c

```diff
@@ -82,40 +81,44 @@ void read_parameterfile(char *fname)
 
     message(0, "----------------------------------------------\n");
     
-    Omega = param_get_double(ps, "Omega0");
+    CP.Omega0 = param_get_double(ps, "Omega0");
+    CP.OmegaLambda = param_get_double(ps, "OmegaLambda");
+    CP.OmegaBaryon = param_get_double(ps, "OmegaBaryon");
+    CP.HubbleParam = param_get_double(ps, "HubbleParam");
+    CP.CMBTemperature = param_get_double(ps, "CMBTemperature");
+    CP.RadiationOn = param_get_double(ps, "RadiationOn");
+    /*This should be removed if massive neutrinos are implemented*/
+    CP.MasslessNeutrinosOn = 1;
```

Comment:
You only add massless neutrinos if there are no massive neutrinos - if MP-GenIC supported adding massive neutrinos to the ICs you would want to set CP.MasslessNeutrinosOn = 0.